examples/fun_mut_const.sou

```diff
@@ -0,0 +1,16 @@
+mut x = 1
+const y = 2
+
+call c1 = take_const (x)
+call c2 = take_const (y)
+call c3 = take_mut (&x)
+
+const s1 = (c1+c2)
+print (s1+c3)
+
+
+function take_const (const a)
+  return a
+
+function take_mut (mut a)
```

Comment:
Show an example of assigning to a mutable variable? (Then again, I think we already have this in a test.)